migration/README.md

```diff
@@ -1,34 +1,44 @@
 # RACK migration tool
 
-NOTE: eventually we can have the tool migrate between different named versions
-of RACK revisions.  This current prototype solely works on updating nodegroups
-from the v4.1 to the v5.0 revision.
+The RACK migration tool allows one to automatically perform most of the updating
+needed for nodegroups when the RACK ontology changes.  Most required changes are
+entirely determined by the nature of the ontology change, and so the migration
+tool can perform these automatically.  Some ontology changes, however, have
+consequences on the nodegroups that are best resolved by an advised human
+operator, and so the tool does not attempt to solve those.
 
 Some changes may have been missed as they were manually listed. If you encounter
 issues with some migration, feel free to reach out to Val (val@galois.com) for
 help.
 
 ## How to build?
 
-To set up, we suggest using Python's virtualenv.  A possible setup is to run:
+To set up, we suggest using Python's virtualenv.  A possible setup is to run
+(replacing `<env>` with the name of the folder you want your virtual environment
+saved in):
 
 ```
-python3 -m venv env       # Creates a virtual environment in the 'env' directory
-source env/bin/activate   # Sets up the virtual environment in the **current** shell
+python3 -m venv <env>       # Creates a virtual environment in the 'env' directory
```

Comment:
I think it makes sense to be general here, but that stops people from copy/pasting as easily and it seems quite rare that people will need multiple virtual environments. I'm OK to leave it like this if you prefer it though.

Reply:
That makes sense, I think the comment makes it explicit enough that `env` is something under your control.